zebedee-cms/src/main/java/com/github/onsdigital/zebedee/ZebedeeConfiguration.java

```diff
@@ -200,6 +195,8 @@ public Path getRedirectPath() {
         return redirectPath;
     }
 
+    public Path getServicePath() { return servicePath;}
```

Comment:
To be consistent with other methods in this file, probably want to multi-line this